docs/graphviz.rst

```diff
@@ -6,6 +6,12 @@ This theme adds several improvements to the built-in :py:mod:`sphinx.ext.graphvi
 - default fonts and colors are set to match the fonts and colors used by the
   theme, including support for both light and dark mode;
 
+  .. admonition:: Only compatible with Google fonts
+
+      When not using a Google font via :themeconf:`font` in conf.py (eg. using a
+      system/custom/self-hosted font), this extension will defer to the Graphviz default font
```

Comment:
Will use the metrics from graphviz's default font , rather than the actual font, for the purpose of computing the graph layout.  However, the text will still be displayed in the browser using the user-specified font.

Reply:
done. It is now:
```rst
      When not using a Google font via :themeconf:`font` in conf.py (eg. using a
      system/custom/self-hosted font), this extension will use the metrics from graphviz's
      default font, rather than the actual font, for the purpose of computing the graph layout.
      However, the text will still be displayed in the browser using the user-specified font.
```